heon/multi_server.py

```diff
@@ -0,0 +1,81 @@
+import socket
+from FlowRecord import FlowRecord
+import argparse
+
+# Define constants
+MAX_CHUNK_SIZE = 100
+
+# Empty previous flow_sequence in flie
+f = open("flow_sequence_server_side.txt", "w")
+f.write("")
+f.close()
+
+# Empty previous ts in file
+f = open("flow_ts_server_side.txt", "w")
+f.write("")
+f.close()
+
+
+# Respond to UDP connections
+def client_handler(ss):
+
+    while True:
+        (data, sender) = ss.recvfrom(MAX_CHUNK_SIZE)
+        print("\n\n---------- CLIENT_HANDLER ----------")
+        print("* FLOW RECEIVED!")
+        print("        * sender => " + str(sender))
+        msg = data.decode('utf-8')
+        print("        * MSG => " + str(msg))
+
+        # Initialize new flow record
+        flow_num = msg
+        flow_record = FlowRecord(flow_num=flow_num)
+
+        # Write flow record to file
+        f = open("flow_sequence_server_side.txt", "a")
```

Comment:
These file names could come from default values for arguments that are available on the command line.

Reply:
It's generally a nice idea to have magic constants (like strings for file names) all be available in one place, so that you can look for them and edit them (or set them from the command line) as one sees fit.

Reply:
Thanks for catching this, I actually forgot to add them to the command line arguments.